backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/receiptschedule/M_ReceiptSchedule_StepDef.java

```diff
@@ -115,13 +115,22 @@ public void there_are_receiptSchedule(final int timeoutSec, @NonNull final DataT
 			final I_C_OrderLine orderLine = orderLineTable.get(orderLineIdentifier);
 
 			final String bPartnerIdentifier = DataTableUtil.extractStringForColumnName(tableRow, COLUMNNAME_C_BPartner_ID + "." + TABLECOLUMN_IDENTIFIER);
-			final I_C_BPartner bPartner = bPartnerTable.get(bPartnerIdentifier);
+			final Integer bPartnerID = bPartnerTable.getOptional(bPartnerIdentifier)
+					.map(I_C_BPartner::getC_BPartner_ID)
+					.orElseGet(() -> Integer.parseInt(bPartnerIdentifier));
+			Assertions.assertThat(bPartnerID).isNotNull();
```

Comment:
i think an Identifier should not be an ID. i can lead to FUD

Reply:
I see what you mean, in some cases it really helped us (to allow that fallback) but I think here we can drop it @dm-ps